app/models/cms/partial.rb

```diff
@@ -19,21 +19,16 @@ def search
   def to_xml(options = {})
     xml = options[:builder] || Nokogiri::XML::Builder.new
 
-    xml.__send__(options.fetch(:root, :partial)) do |x|
+    xml.__send__(CMS::XML::CLASS_TAG.call(self.class)) do |x|
       unless new_record?
         xml.id id
         xml.created_at created_at.xmlschema
         xml.updated_at updated_at.xmlschema
       end
-
       x.system_name system_name
-      x.content_type content_type
-      x.handler handler
-      x.liquid_enabled liquid_enabled
```

Comment:
I guess not relevant for partials?

Reply:
Same, removed for being fixed, empty or not applicable fields for this type.